src/gui/mzroll/scatterplot.cpp

```diff
@@ -121,9 +123,17 @@ void ScatterPlot::setupToolBar() {
 ScatterPlot::~ScatterPlot() { delete(compareSamplesDialog); }
 
 
-void ScatterPlot::setTable(TableDockWidget* t) { 
-	_table=t; 
-	compareSamplesDialog->setTableWidget(t); 
+void ScatterPlot::setTable(TableDockWidget* t) {
```

Comment:
@saifulbkhan better variable name, please

Reply:
Did you mean `TableDockWidget *t`? It can have a better name.